### ISSUE TITLE
CSS files not being watches properly

### ISSUE THREAD

**Issue by jchip** [electrode-io/electrode#567]
moved from https://github.com/electrodejs/electrode-archetype-react-app/issues/51

When a css file that is imported into another css file changes, no updates happen. The file change is not being monitored. But if the same css file is incuded as a dummy css file, the file is watched properly.

// components/special.jsx
import styles from "./special.css";
import {} from "./theme.css"; // to ignore, just to force the monitoring of the file

// special.css
@import "./theme.css";  /* this is not enough for the watcher to watch the theme.css file for changes */

Is this even an issue for electrode-archetype-react-app or some underneath component?

In conclusion, chained css files trough import are not watched, although the css-loader https://github.com/webpack/css-loader/tree/v0.23.1 states:
> @import and url(...) are interpreted like require() and will be resolved by the css-loader. Good loaders for requiring your assets are the file-loader and the url-loader which you should specify in your config (see below).
